webpack/strategies/docs.js

```diff
@@ -23,10 +23,10 @@ export default (config, options) => {
         noParse: /babel-core\/browser/,
         loaders: config.module.loaders
           .map(value => {
-            if (/\.js\/$/.test(value.test.toString())) {
+            if (/js/.test(value.test.toString())) {
```

Comment:
The change in [docs-development](https://github.com/react-bootstrap/react-bootstrap/blob/v0.22.0/webpack/strategies/docs-development.js#L20) is a regex used to test a regex. Whereas this line is a regex to test a file path. This should stay as it is.